File: compiler/bench/over3.fs
\ expected: 300000000
\ Over on depth 3, 300M times
: main
  0 1 2                     ( a b c )
  300000000 0 do
    >r over r>              ( a b a c ) over from depth 3
    drop nip swap           ( a' b c )
    >r >r 1 + r> r>
  loop
  2drop . cr
;

File: compiler/bench/over3.c
// expected: 300000000
// Over on depth 3, 300M times
#include <stdio.h>

int main(void) {
    long a = 0, b = 1, c = 2;
    for (long i = 0; i < 300000000L; i++) {
        long over = a;  // over from depth 3
        a = over + 1;
    }
    (void)b;
    (void)c;
    printf("%ld\n", a);
    return 0;
}
